packages/preview/create-lattes-cv/0.1.0/lib.typ

```diff
@@ -0,0 +1,140 @@
+#import "@preview/datify:0.1.3": *
+#import "src/import.typ": *
+
+// Função lattes_cv: criar o PDF com os dados de lattes
+// Arguments:
+// - database: o arquivo de TOML com os dados de Lattes (string)
+// - kind: o tipo de currículo Lattes (string)
+// - me: o nome para destacar nas citações (string)
+// - last_page: resumo de produção no final (boolean)
+// - data: a data de currículo (datetime)
+// - subtitle: para a página inicial 
+#let lattes-cv(
+  database: "output.toml",
+  kind: "completo", 
+  me: str,
+  last_page: true,
+  date: datetime.today(),
+  subtitle: "Curriculum Vitae",
+  body,
+) = {
+    // define details:
+    let details = toml(database)
```

Comment:
Because files are read relatively to the current package/project when using `toml` (and other functions that read from external files), this won't work as expected: you would only be able to access files in the package directory, not in the user's project. You should take the parsed data directly as a parameter, instead of a file path.